app/assets/javascripts/lib/util/analytics.js

```diff
@@ -0,0 +1,31 @@
+var Analytics = (function () {
+    'use strict';
+
+    function track(obj) {
+        if (window.ga) {
+            window.ga('send', obj.event, obj.type, obj.data);
+        }
+    }
+
+    Analytics.prototype = {
+        chartAdded: function (key) {
```

Comment:
I've noticed that this gets fired every time a chart loads:

* Twice when the page loads for the default chart.
* Every time the user moves to a different slide and the default chart changes.

Maybe we'd be better splitting this into two separate events:

* "chart loaded" – what we have now
* "chart added" – fired when the visitor explicitly selects a new chart

Reply:
In this case I'm okay with just tracking every time it's 'added' and not how many times somebody loads the same chart.